index.js

```diff
@@ -98,22 +98,31 @@ MarketplaceClient.prototype.validatePackage = function(packagePath) {
   return promise;
 };
 
-MarketplaceClient.prototype.publish = function(validatedManifestId) {
+MarketplaceClient.prototype.publish = function(validationId, type) {
+  if (type === null) {
```

Comment:
I guess it'd be better to check the type of the type argument, so it is set to the default value whenever an incompatible type is used.